composer.json

```diff
@@ -12,7 +12,9 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^5.4 | ^7.0"
+        "php": "^5.4 | ^7.0",
+        "psr/container": "^1.0",
+        "dhii/stringable-interface": "^0.1"

```

Comment:
Why require these? Are they *required* for the code to work? Or are they just optionally acceptable?

Reply:
Yes, they are required, because in my code:
```php
use Psr\Container\ContainerInterface;
use Dhii\Util\String\StringableInterface as Stringable;
```

Reply:
Importing an FQN does not make it required.